src/gluonnlp/model/bert.py

```diff
@@ -385,8 +385,12 @@ def forward(self, inputs, token_types, valid_length=None, masked_positions=None)
         This is used in training or fine-tuning a BERT model.
         """
         outputs = []
-        seq_out, _ = self._encode_sequence(inputs, token_types, valid_length)
+        seq_out, attention_out = self._encode_sequence(inputs, token_types, valid_length)
         outputs.append(seq_out)
+
```

Comment:
Could you document this output in line 300 for the attention output?

Reply:
Added extra description